### DISCUSSION TITLE
[Feature] Add actions for assets in Memory on Mobile

### DISCUSSION THREAD

**Discussion by dpantel** [immich-app/immich#4950] [Feature Request]
### The feature

On the Web, you can scroll down from any photo under the Memory pseudo-album to see details and perform actions on it.

On Mobile, you cannot do anything to photos in Memories. I wanted to forward a photo that popped up on my Memory screen today, but was not able to.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4496
